src/material/core/focus_indicator/_focus-indicator.scss

```diff
@@ -0,0 +1,61 @@
+@import '../theming/theming';
+
+@mixin mat-strong-focus-indicators() {
```

Comment:
I think these are the first public mixins we've added since I learned about [SassDoc](http://sassdoc.com). It would be good to add a public-facing description of both of these mixins.

Reply:
Where should this public-facing desc live? Just as a comment above both mixins?

Reply:
Yeah, a comment with three slashes
```scss
/// Applies styles for...
@mixin mat-...() {
}
```